modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -734,14 +734,7 @@
                       "claimType": "Compensation",
                       "closeDate": "2017-10-18",
                       "contentionList": [
-                        "abnormal heart (New)",
-                        " abscess kidney (New)",
-                        " encephalitis lethargica residuals (New)",
-                        " dracunculiasis (New)",
-                        " gingivitis (New)",
-                        " abnormal weight loss (New)",
-                        " groin condition (New)",
-                        " metritis (New)"
+                        "abnormal heart (New), abscess kidney (New), encephalitis lethargica residuals (New), dracunculiasis (New), gingivitis (New), abnormal weight loss (New), groin condition (New), metritis (New)"
```

Comment:
this is what I'm referring to. the response from BGS is a comma separated value, as a string. In the logic above, you're just adding this string to an array. we want to split and then remove leading (and trailing for that matter) whitespace. Each of these should be their own entry within the array

```
"contentList": [
  "abnormal heart (New)",
  "abscess kidney (New)",
  "enceph.....
]
```